### DISCUSSION TITLE
onlyfans scraper

### DISCUSSION THREAD

**Discussion by niemajak900** [UltimaHoarder/UltimaScraper#1079] [Q&A]
Does your onlyfans scrapper program allow to download videos that a creator sends to their subscribers via private messages(including paid ones)? that would be awesome :D

**Comment by TheMissingPort** [comment]
Yes

**Comment by niemajak900** [comment]
How? I don't see anything like that in the tutorial

**Reply by TheMissingPort** [reply]
Well it does lol

**Comment by niemajak900** [comment]
Could you copy and paste it here :D or paste link coz I cant find it anywhere

**Reply by TheMissingPort** [reply]
Copy and paste what

**Comment by niemajak900** [comment]
Description where does it says that this onlyfans scraper allow us to download videos that the creator sends to all other subscribers thru private messages.

**Comment by LunarPenguin** [comment]
Yes the scraper can download videos and picture in messages but only if you could also access them via the webbrowser. The scraper CANNOT download images and videos you have NOT paid for IE: pay-per-view messages
